components/ResidentPage/CaseNoteDialog.spec.tsx

```diff
@@ -138,6 +141,33 @@ describe('CaseNoteDialog', () => {
     expect(screen.getAllByRole('definition').length).toBe(6);
   });
 
+  it('strips html tags from historic case notes', () => {
+    (useRouter as jest.Mock).mockReturnValueOnce({
+      query: {
+        case_note: mockedCaseNote.recordId,
+      },
+    });
+
+    render(
+      <CaseNoteDialog
+        totalCount={1}
+        socialCareId={123}
+        caseNotes={[
+          {
+            ...mockedCaseNote,
+            caseFormData: {
+              ...mockedCaseNote.caseFormData,
+              is_historical: true,
+            },
+          },
+        ]}
+      />
+    );
+
+    expect(screen.getByText('foo historic'));
```

Comment:
Maybe add `.toBeVisible()`?

Reply:
pretty sure they're functionally identical since getByText [throws an error](https://testing-library.com/docs/react-testing-library/cheatsheet/) if the thing isn't found